HISTORY.rst

```diff
@@ -19,6 +19,7 @@ New features and enhancements
 * New ``**indexer`` keyword args added to many indicators, it accepts the same arguments as ``xclim.indices.generic.select_time``, which has been improved. Unless otherwise specified, the time selection is done before any computation. (:pull:`934`, :issue:`899`).
 * Rewrite of ``xclim.sdba.ExtremeValues``, now fixed with a correct algorithm. It has not been tested extensively and should be considered experimental. (:pull:`914`, :issue:`789`, :issue:`790`).
 * Added `days_over_precip_doy_thresh` and `fraction_over_precip_doy_thresh` indicators to distinguish between WMO and ECAD definition of the Rxxp and RxxpTot indices. (:issue:`931`, :pull:`940`)
+* Added `heat_index` indicator.
```

Comment:
```suggestion
* Added `heat_index` indicator. (:issue:`807`, :pull:`915`)
```
It would be good to have a bit more of a description. Maybe mention the source for the methods?